glide.yaml

```diff
@@ -26,9 +27,11 @@ import:
   repo: https://github.com/quantum/goautoneg.git
 - package: github.com/coreos/pkg/capnslog
 - package: k8s.io/client-go
-  version: e121606b0d09b2e1c467183ee46217fa85a6b672
+  version: release-3.0
   subpackages:
   - kubernetes/typed/core/v1
+- package: k8s.io/apimachinery
+  version: release-1.6
```

Comment:
is this dependency on 1.6 going to work in our current coreos-kubernetes vagrant cluster that uses 1.5.4?